core/sqf/src/seatrans/hbase-trx/src/main/java/org/apache/hadoop/hbase/client/transactional/SsccTransactionalTable.java

```diff
@@ -274,6 +278,48 @@ public SsccDeleteTransactionalResponse call(SsccRegionService instance) throws I
               throw new IOException(resultArray[0].getException());
           }
 
+    public void deleteRegionTx(final long tid, final Delete delete, final boolean autoCommit) throws IOException {
+      SingleVersionDeleteNotSupported.validateDelete(delete);
+      final String regionName = super.getRegionLocation(delete.getRow()).getRegionInfo().getRegionNameAsString();
+         Batch.Call<SsccRegionService, SsccDeleteRegionTxResponse> callable =
+                new Batch.Call<SsccRegionService, SsccDeleteRegionTxResponse>() {
+            ServerRpcController controller = new ServerRpcController();
+            BlockingRpcCallback<SsccDeleteRegionTxResponse> rpcCallback =
+            new BlockingRpcCallback<SsccDeleteRegionTxResponse>();
+
+            public SsccDeleteRegionTxResponse call(SsccRegionService instance) throws IOException {
```

Comment:
Good comment.  I don't think @Override is needed on any of the region operations since none are present in the super class